movie.go

```diff
@@ -78,6 +84,18 @@ func getMovieWithDetail(id int) MovieDetails {
 				} else {
 					var actors Actors
 					json.Unmarshal(body, &actors)
+
+					var crew Crew
+					json.Unmarshal(body, &crew)
+
+					for i := 0; i < len(crew.EntireCrew); i++ {
+						fmt.Println(crew.EntireCrew[i].Name + " " + crew.EntireCrew[i].Job)
+						if crew.EntireCrew[i].Job == "Producer" || crew.EntireCrew[i].Job == "Executive Producer" {
```

Comment:
Are there any movies in the API that have both an Executive Producer and a Producer? Or movies with multiple Producers? I don't know if there's any best way to distinguish between multiple producers other than to show the first listed, but I think there should certainly be a check for Executive Producer before you look for all producers.

Reply:
After searching a few fields it appears that Execute producer always appears before producer. Some movies also don't have an executive producer but instead only have a producer. This code ensures the most important producer is always pulled.